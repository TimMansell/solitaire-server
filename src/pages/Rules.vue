<template>
  <GameOverlay show-close data-test="rules-overlay">
    <template #title> How to play </template>
    <template #msg>
      <Rules />
    </template>
  </GameOverlay>
</template>

<script>
import GameOverlay from '@/components/GameOverlay.vue';
import Rules from '@/components/Rules.vue';

export default {
  name: 'RulesPage',
  components: {
    GameOverlay,
    Rules,
  },
};
</script>
