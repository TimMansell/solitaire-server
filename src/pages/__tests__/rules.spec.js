import { shallowMount } from '@vue/test-utils';
import Rules from '@/pages/Rules.vue';

describe('Rules.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Rules);

    expect(wrapper).toMatchSnapshot();
  });
});
