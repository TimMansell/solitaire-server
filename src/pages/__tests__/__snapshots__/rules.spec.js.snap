// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules.vue matches snapshot 1`] = `
<gameoverlay-stub showclose="true" data-test="rules-overlay"> How to play <rules-stub></rules-stub>
</gameoverlay-stub>
`;
